ruby/red-arrow/ext/arrow/arrow.cc

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2018 Kenta Murata <mrkn@mrkn.jp>
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "red_arrow.hpp"
+
+namespace red_arrow {
+
+VALUE mArrow;
```

Comment:
I personally prefer such style so I fixed the indentation.

The reason why I didn't indent namespaces is that I followed the style of Arrow C++.

Reply:
OK. We indent in Ruby.

FYI: Arrow C++ uses Google based cording style. But Arrow GLib doesn't follow it. Because I want to provide headers for C and C++ of the same object such as `arrow-glib/decimal128.h` and `arrow-glib/decimal128.hpp`. But Google's style use `.h` for all header files.
Red Arrow may provide header files for C and C++. So we can't use Google's style.

Reply:
I've understood the situation.